Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -1196,6 +1197,11 @@ void MenuBar::GenerateSymbolsFromSignatureDB()
 
 void MenuBar::GenerateSymbolsFromRSO()
 {
+  int ret =
+      ModalMessageBox::question(this, tr("RSO auto-detection"), tr("Auto-detect RSO modules?"));
```

Comment:
I wonder if this is the best string to be presented to a user...but then again, this seems pretty advanced and whoever uses it likely knows what RSO means.
Yet, would it hurt to leave an `i18n` comment for translators with the spelled out wording for RSO?